.circle/tests.sh

```diff
@@ -46,4 +46,8 @@ case ${CIRCLE_NODE_INDEX} in
     fmriprep-docker -i poldracklab/fmriprep:latest --config $HOME/nipype.cfg -w $HOME/ds005/scratch $HOME/data/ds005 $HOME/ds005/out participant --output-space fsaverage5 --debug --write-graph
```

Comment:
Just add `--use_aroma` to this line. We don't need to add another container. If you're going to need outputs in MNI space, just remove the `--output-space fsaverage5` part.

You don't need to set the template; that's the default.